docs/general/get-involved.md

```diff
@@ -179,4 +199,4 @@ Special error codes.
 ````
 
 For real example see [/user](../backend-api/resources/commons/user/index.md) and
-[:octicons-file-code-24: source](https://raw.githubusercontent.com/SquareGPS/navixy-api/master/docs/backend-api/resources/commons/user/user.md).
\ No newline at end of file
+[:octicons-file-code-24: source](..commons/user/index.md).

```

Comment:
No... I did not write the full path in previous review. I only meant that at the end you need to change `used.md` to `index.md`. Please, check links before committing.

Reply:
was confused because ..commons/user/index.md was eascaped with ` `